packages/paste-core/components/button/src/types.ts

```diff
@@ -29,6 +30,17 @@ export const DirectButtonPropTypes = {
   type: PropTypes.oneOf(['submit', 'button', 'reset']) as any,
   disabled: PropTypes.bool,
   buttonState: PropTypes.oneOf(['disabled', 'loading', 'default']).isRequired as any,
+  variant: PropTypes.oneOf([
```

Comment:
Created a ticket to refactor these as a map like we do with other components, i.e. Alert, Anchor, etc.